package.json

```
@@ -14,14 +14,14 @@
 		"node": ">=14.18"
 	},
 	"scripts": {
-		"create-rule": "node ./scripts/create-rule.mjs && npm run generate-rule-notices && npm run generate-rules-table",
+		"create-rule": "node ./scripts/create-rule.mjs && npm run fix:eslint-docs",
 		"fix": "run-p --continue-on-error fix:*",
+		"fix:eslint-docs": "eslint-doc-generator --ignore-deprecated-rules --ignore-config all --rule-doc-title-format desc --url-configs \"https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs\"",
 		"fix:js": "npm run lint:js -- --fix",
 		"fix:md": "npm run lint:md -- --fix",
-		"generate-rule-notices": "node ./scripts/generate-rule-notices.mjs",
-		"generate-rules-table": "node ./scripts/generate-rules-table.mjs",
 		"integration": "node ./test/integration/test.mjs",
 		"lint": "run-p --continue-on-error lint:*",
+		"lint:eslint-docs": "npm-run-all \"fix:eslint-docs --check\"",
```

Is `npm-run-all` needed here?

---

`npm-run-all` allows us to call another NPM script without having to specify either `npm` or `yarn` or whatever package manager the user might be using.

---

```suggestion
		"lint:eslint-docs": "npm run fix:eslint-docs -- --check",
```

Make sense to use npm run, like other scripts did.